packages/api/src/controllers/asset.ts

```diff
@@ -980,6 +980,10 @@ app.post(
           url: downloadUrl,
           catalystPipelineStrategy: catalystPipelineStrategy(req),
           encryption,
+          thumbnails: !(await isExperimentSubject(
+            "vod-thumbs-off",
+            req.user?.id,
+          )),
```

Comment:
If you want to have this logic for all assets, might wanna check usages of this `createTask` function. I know there's at least recordings tasks which are created from `cannon.ts`